.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+# Description
+Briefly describe the problem or reason for the changes implemented in the PR
+
+# Trac ticket
+Link to the Trac ticket for this PR (all PRs except really trivial issues or 
+tests adding need an accepted ticket).
+
+# UI Changes
+Please attach screenshots in light and dark mode for UI changes.
+
```

Comment:
Let’s put this as a final point in the checklist. It doesn’t need a separate section.

```
- [ ] For UI changes, I have attached screenshots in both light and dark modes.
```